CHANGELOG.md

```diff
@@ -3,6 +3,51 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [2.34.0]
+### Added
+- Approvals Bootstrap Task
```

Comment:
"Included a new approvals saga implemented by the Approvals Bootstrap Task, an example on how to configure it is found below. Please be aware that this saga is not idempotent due to some product limitations. Mor information in the wiki page <link to wiki>"

Also please include the link for the tag reference at the bottom of this file.